test/src/test/java/org/corfudb/runtime/clients/LayoutClientTest.java

```diff
@@ -137,6 +141,26 @@ public void proposeRejectsAlreadyProposed()
     }
 
     @Test
+    public void proposeMalformedRejected()
+            throws Exception {
+        Layout layout = TestLayoutBuilder.single(9000);
+        long epoch = layout.getEpoch();
+        assertThat(client.bootstrapLayout(layout).get())
+                .isEqualTo(true);
+
+        assertThat(client.prepare(epoch, 10L).get() != null)
```

Comment:
I would prefer if we started already (same as in other recent PRs). Not a big deal, but I prefer not to use 10L.